packages/sui-studio/src/components/demo/try-require.js

```diff
@@ -2,18 +2,15 @@
 
 const reqComponentsSrc =
   require.context(`bundle-loader?lazy!${__BASE_DIR__}/components`, true, /^\.\/\w+\/\w+\/src\/index\.jsx?/)
-const reqComponentsPkg =
-  require.context(`bundle-loader?lazy!${__BASE_DIR__}/components`, true, /^\.\/\w+\/\w+\/package\.json?/)
-const reqComponentsPlayGround =
-  require.context(`bundle-loader?lazy!raw-loader!${__BASE_DIR__}/demo`, true, /^.*\/playground/)
-const reqContextPlayGround =
-  require.context(`bundle-loader?lazy-loader!${__BASE_DIR__}/demo`, true, /^.*\/context\.js/)
-const reqRouterPlayGround =
-  require.context(`bundle-loader?lazy-loader!${__BASE_DIR__}/demo`, true, /^.*\/routes\.js/)
-const reqEventsPlayGround =
-  require.context(`bundle-loader?lazy-loader!${__BASE_DIR__}/demo`, true, /^.*\/events\.js/)
 
-const tryRequire = ({category, component}) => {
+const requireFile = async ({ defaultValue, importFile }) => {
+  const file = await importFile().catch(_ => defaultValue)
+  return typeof file === 'undefined'
+    ? Promise.reject(new Error('Error requiring file'))
+    : file.default
+}
+
+const tryRequire = async ({category, component}) => {
   const exports = new Promise(resolve => {
```

Comment:
Why didn`t you migrate the require of the component to the new promises API?!

Reply:
I had some problems!!!!?1'1'11!! xD And I thought to do it later but as you asked, I managed to fix them. :) Now it works. ;)